example/Fixtures/Syntax/index.js

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { lowerCase } from 'lodash';
+
+import callouts from './callouts.md';
+import codeBlockTests from './code-blocks/code-block-tests.md';
+import codeBlockVarsTest from './code-blocks/vars-test.md';
+import codeBlocks from './code-blocks.md';
+import embeds from './embeds.md';
+import features from './features.md';
+import headings from './headings.md';
+import images from './images.md';
+import lists from './lists.md';
+import tables from './tables.md';
+
+const fixtures = Object.entries({
+  codeBlocks,
+  codeBlockTests,
+  codeBlockVarsTest,
+  callouts,
+  embeds,
+  tables,
+  lists,
+  headings,
+  images,
+  features,
+}).reduce((memo, [sym, doc]) => {
+  const name = lowerCase(sym);
```

Comment:
Any reason to not use vanilla JS here? Then we can nix Lodash!
```suggestion
  const name = sym;
```

Reply:
No, I can write my own lowerCase.

Reply:
Just use the [built-in **`string.toLowerCase()`**](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase)! (Sorry, I forgot to paste it in to my suggestion!)

Reply:
I want `lowerCase` => `lower case`.

Reply:
Ohhhhhh shit, guess I shoulda read the lodash docs before I commented....

Reply:
In that case, feel free to leave the dependency.